app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt

```diff
@@ -185,29 +193,47 @@ fun HikeDetailScreen(
   val hikeLineColor = route.getColor()
   MapUtils.showHikeOnMap(mapView = mapView, hike = route, color = hikeLineColor, onLineClick = {})
 
-  Box(modifier = Modifier.fillMaxSize().testTag(Screen.HIKE_DETAILS)) {
-    // Map
-    AndroidView(
-        factory = { mapView },
-        modifier =
-            Modifier.fillMaxWidth()
-                .padding(bottom = 300.dp) // Reserve space for the scaffold at the bottom
-                .testTag(TEST_TAG_MAP))
-    // Back Button at the top of the screen
-    BackButton(
-        navigationActions = navigationActions,
-        modifier = Modifier.padding(top = 40.dp, start = 16.dp, end = 16.dp),
-        onClick = { listOfHikeRoutesViewModel.clearSelectedRoute() })
-    // Zoom buttons at the bottom right of the screen
-    ZoomMapButton(
-        onZoomIn = { mapView.controller.zoomIn() },
-        onZoomOut = { mapView.controller.zoomOut() },
-        modifier =
-            Modifier.align(Alignment.BottomEnd)
-                .padding(bottom = MapScreen.BOTTOM_SHEET_SCAFFOLD_MID_HEIGHT + 8.dp))
-
-    // Hike Details bottom sheet
-    HikeDetails(detailedRoute, savedHikesViewModel, elevationData)
+  LaunchedEffect(Unit) {
+    if (authViewModel.currentUser.value == null) {
```

Comment:
Question: Is this case actually possible? If such wouldn't it be more appropriate to show a toast?

Reply:
It should never happen. This is just a sanity check to prevent NullPointerException

Reply:
Ok yes that seems more logical